software/core/oqm-core-characteristics/makeInstallers.sh

```
@@ -0,0 +1,154 @@
+cat <<EOT >> "$buildDir/$debDir/DEBIAN/postrm"
+#!/bin/bash
+
+systemctl daemon-reload
+# Remove docker image
+if [[ "$(docker images -q oqm-core-base_station 2> /dev/null)" != "" ]]; then
+        docker rmi oqm-core-base_station
+        echo "Removed docker image."
+else
+        echo "Docker image was already gone."
+fi
+if [ $( docker ps -a | grep oqm-core-base_station | wc -l ) -gt 0 ]; then
```

Shellcheck (0.11.0)

[style] Consider using 'grep -c' instead of 'grep|wc -l'.

(SC2126)